src/scrabbleScore.js

```diff
@@ -24,15 +24,25 @@ let SCRABBLE_TILE_SCORES = {
  */
 
 function scrabbleScore(word) {
-  // This is your job. :)
+  let score = 0;
+  let keys = Object.keys(SCRABBLE_TILE_SCORES);
+  for (let letter of word) {
+    letter = letter.toLowerCase();
+    for (key of keys) {
```

Comment:
The only reason I can think of to iterate through `keys` just to see if it contains`letter` is to do something special if it's not present.

But if `letter` is not in `keys` then `SCRABBLE_TILE_SCORES[letter]` would be `undefined`, in which case there's no reason to iterate through keys at all.